wasmtime/bindgen/__init__.py

```diff
@@ -1,28 +1,106 @@
-from .generated import Bindgen, BindgenImports, Err
-from typing import Mapping, Tuple
+from .generated import Root, RootImports, Err
+from .generated.imports import Exit
+from .generated.imports import Random
+from .generated.imports import Stdin
+from .generated.imports import Stdout
+from .generated.imports import Stderr
+from .generated.imports import streams
+from .generated.imports import Preopens
+from .generated.imports import Environment
+from .generated.imports import Filesystem
+from .generated.imports.filesystem import Descriptor, Filesize, ErrorCode, DescriptorType
+from .generated import types as core_types
+from typing import Mapping, Tuple, List
+
 import sys
 from wasmtime import Store
+from Crypto.Random import get_random_bytes
```

Comment:
Could this perhaps use [`os.urandom`](https://docs.python.org/3/library/os.html#os.urandom) instead of picking up a new dependency?

Reply:
Yeah that would be much better. I'll change this.